tests/chaos/oadp/conftest.py

```diff
@@ -0,0 +1,66 @@
+import datetime
+import logging
+
+import pytest
+
+from utilities.constants import BACKUP_STORAGE_LOCATION, FILE_NAME_FOR_BACKUP, TEXT_TO_TEST, TIMEOUT_3MIN, TIMEOUT_10MIN
+from utilities.infra import ExecCommandOnPod, wait_for_node_status
+from utilities.oadp import VeleroBackup, create_rhel_vm
+from utilities.storage import write_file
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def rhel_vm_with_dv_running(request, admin_client, chaos_namespace, snapshot_storage_class_name_scope_module):
+    """
+    Create a RHEL VM with a DataVolume.
+    """
+    vm_name = request.param["vm_name"]
+
+    with create_rhel_vm(
+        storage_class=snapshot_storage_class_name_scope_module,
+        namespace=chaos_namespace.name,
+        vm_name=vm_name,
+        dv_name=f"dv-{vm_name}",
+        client=admin_client,
+        wait_running=True,
+        rhel_image=request.param["rhel_image"],
+    ) as vm:
+        write_file(
+            vm=vm,
+            filename=FILE_NAME_FOR_BACKUP,
+            content=TEXT_TO_TEST,
+            stop_vm=False,
+        )
+        yield vm
+
+
+@pytest.fixture()
+def oadp_backup_in_progress(chaos_namespace, rhel_vm_with_dv_running):
+    timestamp = datetime.datetime.now().strftime("%Y%m%d-%H%M%S")
+    backup_name = f"backup-{timestamp}"
+
+    with VeleroBackup(
```

Comment:
client is missing

Reply:
`client` is `None` in the `VeleroBackup` class, why do I need it here?

Reply:
you need to pass client for every resource that we use
in a previous comment i mentioned that client cannot be None
it will soon be mandatory in the wrapper